src/main/java/fish/payara/sample/debugging/service/LibraryEndpoints.java

```diff
@@ -48,4 +48,11 @@ public Book addBookToLibrary(String name, Book book) {
         return book;
     }
 
+    @Override
+    public Library deleteLibrary(String name) {
+        Library library = getLibrary(name);
+        library.getBooks().remove(library);
```

Comment:
Here a library object is being removed from a list of books. It should be removing the library from the list of libraries.
```suggestion
        model.getLibraries().remove(library);
```